docs/Basic-Guides.md

```diff
@@ -0,0 +1,141 @@
+# Basic Guides
+
+This guide shows you the basic steps you can try to play around with ML-agents. 
+
+If you are not familiar with the [Unity Engine](https://unity3d.com/unity),
+we highly recommend the [Roll-a-ball tutorial](https://unity3d.com/learn/tutorials/s/roll-ball-tutorial) to learn all the basic concepts of Unity. 
+
+## Setting up ML-Agents within Unity
+
+1. Open Unity and use it to open folder `ml-agents/unity-environment`. 
+2. Go to `Edit` -> `Project Settings` -> `Player`
+3. For **each** of the platforms you target 
+(**`PC, Mac and Linux Standalone`**, **`iOS`** or **`Android`**):
+    1. Go into `Other Settings`.
+    2. Select `Scripting Runtime Version` to 
+    `Experimental (.NET 4.6 Equivalent)`
+    3. In `Scripting Defined Symbols`, add the flag `ENABLE_TENSORFLOW`. 
+    After typing in the flag name, press Enter.
+4. Go to `File` -> `Save Project`
+
+![Project Settings](images/project-settings.png)
+
+## Setting up TensorFlowSharp Support
+
+[Download](https://s3.amazonaws.com/unity-ml-agents/0.3/TFSharpPlugin.unitypackage) TensorFlowSharp plugin, and import it into Unity once downloaded by double clicking on it.  You can see if it was successfully imported by checking the TensorFlow files in the Project window under `Assets` -> `ML-Agents` -> `Plugins` -> `Computer`. 
```

Comment:
add "the" after Download

Reply:
Re-write the first sentence as: [Download](https://s3.amazonaws.com/unity-ml-agents/0.3/TFSharpPlugin.unitypackage) the TensorFlowSharp plugin. Then import it into Unity by double-clicking the downloaded file.

Reply:
"You can see if it was" --> "You can check if it was"